src/gondi.ts

```diff
@@ -794,7 +794,7 @@ export class Gondi {
         throw new Error(`No native bid for price ${price}`);
       }
 
-      executionData = await this.reservoir.generateMatchOrdersExecutionData({
```

Comment:
Delete `generateMatchOrdersExecutionData` then

Reply:
match orders is still used for bnpl